design/workflow.go

```diff
@@ -186,8 +186,8 @@ var _ = Service("workflow", func() {
 		HTTP(func() {
 			DELETE("/workflows/assignments/{name}")
 			Param("name")
-			Param("codesetName")
 			Param("codesetProject")
+			Param("codesetName")
 			Response(StatusNoContent)
```

Comment:
Hehe, I reckon you did all this just so you would have the payload builder arguments generated in the right order :))) I appreciate the attention to detail.